atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionSchemaVersionEnforcement.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.impl;
+
+import com.palantir.atlasdb.internalschema.TransactionSchemaManager;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.logger.SafeLogger;
+import com.palantir.logsafe.logger.SafeLoggerFactory;
+import com.palantir.timestamp.ManagedTimestampService;
+
+public final class TransactionSchemaVersionEnforcement {
```

Comment:
A bit awful but probably more reasonable than just writing the entries manually into the coordination table?

Reply:
That approach would be acceptable I guess, because it's a test utility, but indeed I think this is more reasonable